docs/getting-started-guides/docker-multinode.md

```diff
@@ -74,54 +78,165 @@ This pattern is necessary because the `flannel` daemon is responsible for settin
 all of the Docker containers created by Kubernetes.  To achieve this, it must run outside of the _main_ Docker daemon.  However,
 it is still useful to use containers for deployment and management, so we create a simpler _bootstrap_ daemon to achieve this.
 
-You can specify k8s version on very node before install:
+## Starting a Cluster
 
-```
+An example cluster is listed below:
+
+| IP Address  |   Role   |
+|-------------|----------|
+|10.10.102.152|   node   |
+|10.10.102.150|  master  |
+
+Optional, you can specify version before deployment, otherwise, we'll use latest `hyperkube` release as default.
```

Comment:
Bad declaration, huh? Maybe we should list a default version for `hyperkube`, and that's all.